oag/ja-jp/content/topics/access-gateway/configure-idp-okta.htm

```diff
@@ -324,27 +363,56 @@ <h2><span class="mc-variable okta-feature-names.Access_Gateway variable">Access
+                                                    <p>
+                                                        <p class="featureBanner">早期アクセスリリース</p>
+                                                    </p>
+                                                    <ol>
+                                                        <li value="1"><a href="https://auth0.com/docs/get-started/auth0-overview/create-applications/configure-an-identity-provider-in-access-gateway" target="blank" title="Access Gateway内のIDプロバイダーを構成する" alt="Configure an Identity Provider in Access Gateway">Customer Identity Cloudのドキュメント</a>に記載されているプロシージャを実行します。</li>
+                                                        <li value="2"><a href="#ConfigAuth0IdP" class="MCXref xref">Customer Identity CloudをAccess Gateway向けのIdPとして使用する</a>プロシージャを続けます。</li>
                                                     </ol>
                                                     <h2><span class="mc-variable okta-feature-names.Access_Gateway variable">Access Gateway</span>にIdPを構成する</h2>
+                                                    <p><span class="mc-variable okta-feature-names.Access_Gateway variable">Access Gateway</span>向けのIdPには、Workforce Identity CloudとCustomer Identity Cloudのいずれかを利用できます。</p>
+                                                    <h3><a name="ConfigOktaIdP"></a>Workforce Identity Cloudを<span class="mc-variable okta-feature-names.Access_Gateway variable">Access Gateway</span>向けのIdPとして使用する</h3>
                                                     <ol>
-                                                        <li value="1">ブラウザーで、<span class="mc-variable okta-feature-names.Access Gateway_UI variable">Access Gateway 管理者 UI コンソール</span>に移動し、管理者としてさサインインします。 </li>
-                                                        <li value="2"><span class="wintitle">[Settings（設定）]</span>タブを選択します。</li>
-                                                        <li value="3"><span class="wintitle">[Identity Providers（IDプロバイダー）]</span>ペインをクリックします。</li>
-                                                        <li value="4"><span class="uicontrol">［+］</span>をクリックし、<span class="uicontrol">［OKTA］</span>を選択します。</li>
-                                                        <li value="5">以下の情報を入力します。
-								<ul><li><span class="uicontrol">名前</span>：IdPにとって意味のある名前を入力します（例えば、Okta IdP）。</li><li><span class="uicontrol">Okta Org</span>：org名（例えば、<span class="userinput">orgname.oktapreview.com</span>、<span class="userinput">orgname.okta.com</span>など）を入力します。</li><li><span class="uicontrol">Okta APIトークン</span>：Okta APIトークンを作成したら、Okta orgからコピーしたトークをペーストします。</li></ul></li>
+                                                        <li value="1">ブラウザーで<span class="mc-variable okta-feature-names.Access Gateway_UI variable">Access Gateway管理者UIコンソール</span>に移動し、管理者としてサインインします。 </li>
+                                                        <li value="2"><span class="wintitle">[設定]</span>タブを選択します。</li>
```

Comment:
@pengyao-okta here's another `wintitle` attribute that regressed. Interestingly, the next line has a `wintitle` attr that keeps the EN (JA) pattern.